cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -165,6 +170,27 @@ protected Future<Boolean> delete(Reconciliation reconciliation) {
         }).map(ignored -> Boolean.FALSE);
     }
 
+    @Override
+    public void reconcileAll(String trigger, String namespace, Handler<AsyncResult<Void>> handler) {
+        super.reconcileAll(trigger, namespace, handler);
+
+        connectorOperator.listAsync(namespace, selector())
```

Comment:
The selector is not applied on the Connector resources but on the connect resources IIRC. So this would need to select all, query the Connects and check their selectors. Which would be IMHO fairly expensive, but not sure there isa better solution.

Reply:
I've used selector match expression and select connectors by all connect names.